platform/ios/src/MGLMapView.mm

```diff
@@ -1899,9 +1899,13 @@ - (void)resetNorthAnimated:(BOOL)animated
 
 - (void)resetPosition
 {
-    MGLMapCamera *camera = [MGLMapCamera camera];
-    camera.altitude = MGLAltitudeForZoomLevel(0, 0, 0, self.frame.size);
-    self.camera = camera;
+    CGFloat pitch = _mbglMap->getDefaultPitch();
+    CLLocationDirection heading = mbgl::util::wrap(_mbglMap->getDefaultBearing(), 0., 360.);
```

Comment:
It looks like studio wraps bearings. Do we do this since we don't know where the style will actually come from? (just curious)